### ISSUE TITLE
The guides are missing a title attribute in the front matter

### ISSUE THREAD

**Issue by lauracowen** [OpenLiberty/guides-common#425]
The HTML title of the page, which is shown in the browser tab, does not match the heading/title of the guide. This is confusing from a user perspective when trying to find the right browser tab after switching away to a different tab:

![image](https://user-images.githubusercontent.com/3322532/79457381-0e765e00-7fe8-11ea-91df-ae9cf96129c5.png)

There needs to be a `title` or `page-title` line in the front matter of the files like we have in the blog posts, eg https://raw.githubusercontent.com/OpenLiberty/blogs/master/publish/2020-04-15-prometheus-alertmanager-rhocp-open-liberty.adoc

This would enable the seo-title to be different from the guide's main heading (which I assume was intentional) without making the browser tab (HTML title) different from the guide's main heading.

**Comment by yeekangc** [comment]
@mbroz2 @steven1046 @gkwan-ibm

**Comment by steven1046** [comment]
We would have to make an update to the UI code if we want to use a new `title` attribute in guides to change this title, which is generated by asciidoctor right now based on the asciidoc "document title":
![image](https://user-images.githubusercontent.com/6392944/79462566-4916d100-7fbd-11ea-89cb-ea98c327dd77.png)

**Comment by steven1046** [comment]
The title shown here:
![image](https://user-images.githubusercontent.com/6392944/79462566-4916d100-7fbd-11ea-89cb-ea98c327dd77.png)
is already based on the page.title which is not specified in guides currently (they only specify the asciidoctor "document title") so it should work automatically without any UI changes if we add that attribute.

**Comment by steven1046** [comment]
I don't think we can make the browser's tab show something different than what's in the seo-title, because we want the seo-title to be the default <title> set if the page has an seo-title which is what will be shown on search engine results. If it's missing from the page then it will use the page.title to display what's in the browser tab and also if a user sees a page on google. As far as i know, there's no way to make the browser's tab show something different than the page <title> which we want to be the seo title for search engines if the page has an seo-title. @mbroz2 @lauracowen

**Comment by lauracowen** [comment]
I think the point of SEO titles (in general I mean, not in our asciidoc tags) is to be able to show something different (eg more concise) than what's in the article heading/title (https://yoast.com/help/optimizing-the-seo-title-and-meta-description-of-your-homepage/). In general, I would agree with you that they make sense to be the same as the article heading/title - they just don't need to be.

**Comment by steven1046** [comment]
The text in the browser's tab is the seo title. Those have to be the same since they both come from the <title> tag so we can't customize one and not the other. The text "Creating a restful web service" in the middle column could be changed to match those but then we would have a very verbose title. Right now that title comes from the ascidoc document title (The `=` line) and that is also used on the /guides page on the guide card.

https://searchengineland.com/writing-html-title-tags-humans-google-bing-59384

**Comment by lauracowen** [comment]
Sorry, I'm not sure I'm following exactly (I think I'm generally agreeing with you but I don't understand exactly what you just said :) ):

- The `=` line is actually the HTML <h1> tag (Level 1 heading) but I think you're saying it automatically converts to populate the HTML <title> tag too?
- What do browsers use by default in their tabs? I thought that was historically the HTML <title> tag? If that's the case, I don't understand why the browser tab looks different from the `=` heading (name) of the guide.
- What does `seo-title` in the asciidoc front matter of our guides and blog posts become in HTML? And, in our blog posts, what does the `title` tag populate when transformed to HTML?

**Comment by steven1046** [comment]
The H1 heading does come from the = line and not the <title> tag. We insert the `seo-title` into the `<title>` tag in both the blogs and guides, and that is shown on Google and in the browser tab so you're correct about it using the `<title>`. 

The blogs h1 heading comes from the `:page-title:` and so does the guides, however since the guides don't have a `:page-title:` then the h1 heading is using the `=` document title.

**Comment by lauracowen** [comment]
ok.

The blogs don't have a `:page-title:` either but does have a `title:` - does that resolve to the same thing? So if I removed the `=` line from the blog posts, it would still have a H1 heading in the blog post because there's a `title:` line (I always set them to the same thing anyway so I wouldn't have noticed!)?

**Comment by steven1046** [comment]
I tested them without a `title:` and they work.

**Comment by yeekangc** [comment]
@gkwan-ibm, where are we with this?  Thanks.

**Comment by gkwan-ibm** [comment]
@yeekangc tried `:page-title:` and `title:` , still cannot find out a solution
